### ISSUE TITLE
Region overlay when viewing photos

### ISSUE THREAD

**Issue by Terrance** [deckerst/aves#640]
**Is your feature request related to a problem? Please describe.**

I recently noticed the detailed metadata display on the info screen -- very handy!

In particular, I can see face tag regions are detected, and it would be neat if these could be shown on the picture itself.

**Describe the solution you'd like**

When viewing a picture full-sceen, a toggle button in the bottom toolbar to show or hide regions and their corresponding names, something like [how Digikam displays face tags](https://userbase.kde.org/Digikam/Tutorials/Tagging_and_Face_Tags#Face_Tags_%E2%80%93_How_To)?

**Comment by deckerst** [comment]
Related:
- #69

Baby steps:
1) show existing metadata face regions in the viewer
2) index existing metadata face regions in the collection
3) allow manual metadata face region editing
4) automatic face region identification

If I understood correctly, your request is step 1, and #69 is step 2.

**Comment by Terrance** [comment]
Yes, I'm looking to effectively have some labels on faces in the viewer, which looking at the discussion in #69 seems to be a duplicate of #199 (but that was rolled into #69).

2 is less pressing, for me at least, because Digikam already creates associated `People/<name>` tags for each tagged face, so they're already searchable in my collection.

**Comment by enrique-lozano** [comment]
Having this functionality would be amazing!  I hope it is implemented soon.

I am tagging the faces of my entire gallery with DigiKam and this app is very helpful to me when it comes to viewing my gallery in a much more visual way, so thank you very much ♥️
